packages/backend/src/accounting/service.ts

```diff
@@ -123,7 +129,8 @@ export function createAccountingService(
 
 export async function createLiquidityAccount(
   deps: ServiceDependencies,
-  account: LiquidityAccount
+  account: LiquidityAccount,
+  accTypeCode?: AccountTypeCode
```

Comment:
`createLiquidityAccount` tests could be updated to check that the resulting account has the specified code.
I'm also wondering if `code` could just be added to the `LiquidityAccount` interface.

Reply:
Or maybe we should ditch setting the `code` until we're using it.